### PR TITLE
aergocli: deploy contract in hex format

### PR DESCRIPTION
**Pull request by kroggen**
Allow `aergocli` to accept contract code encoded in hex format.

This is used to deploy a copy of an existing contract to another chain, or even the same. Also used for test purposes, when deployed locally.

The data in hex format is copied from aergoscan view of the transaction that deployed the contract.

With this change, the payload given to `aergocli` on `contract deploy` can be either in hex format or the format generated by `aergoluac --payload`

### CODE REVIEW

File: cmd/aergocli/cmd/contract.go [hunk 1]
```diff
@@ -128,6 +129,15 @@ func init() {
 	rootCmd.AddCommand(contractCmd)
 }
 
+func isHexString(s string) bool {
+	for _, c := range s {
```
Comment (hayarobi) [inline cmd/aergocli/cmd/contract.go:133]: It can be more safe by checking the length of string is even

Comment (kroggen) [reply cmd/aergocli/cmd/contract.go:133]: But the output of `aergoluac` can also be even...

Comment (kroggen) [reply cmd/aergocli/cmd/contract.go:133]: Oh, do you mean only inside the `isHexString()` function? Makes sense

**Comment by kroggen** [comment]
Should the base be changed to `develop`?

**Comment by hayarobi** [comment]
I think this PR should be merged to develop for now.
Anyway, it looks safe to merge develop into release since the develop doesn't have any changes that affecf behaviors of aergosvr binary, so I want to merge develop and release/2.4.4 if you agree.

**Comment by kroggen** [comment]
OK, I changed the base to `develop`. You can make a new review or merge this.

It triggered new test.

I suggest to merge the other PR I sent too (only tests)

Then later you can merge develop into release/2.4.4

**Review by hayarobi — APPROVED** [review]
lgtm

**Pull request merged**